src/DonationForms/Actions/ConvertDonationFormBlocksToFieldsApi.php

```diff
@@ -131,6 +132,13 @@ protected function createNodeFromBlockWithUniqueAttributes(BlockModel $block, in
             case "givewp/donor-name":
                 return $this->createNodeFromDonorNameBlock($block);
 
+            //TODO: give_is_donor_comment_field_enabled() prevents this from showing up in admin
```

Comment:
So what are we supposed to do with this? What's the to-do exactly?

Reply:
it was just for a reminder.  for v2 forms you can enable/disable on the form which would open this gate. on v3 forms we only have the global enabled - so would likely need to hook into the form schema to see if the block exists and update the form meta.

Reply:
Resolved 940cbf91